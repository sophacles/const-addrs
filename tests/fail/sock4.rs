use const_addrs::sock4;

fn main() {
    let _ = sock4!("192.168.11:300");
}
