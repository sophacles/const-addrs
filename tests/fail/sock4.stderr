error: invalid IPv4 socket address syntax
 --> tests/fail/sock4.rs:4:20
  |
4 |     let _ = sock4!("192.168.11:300");
  |                    ^^^^^^^^^^^^^^^^
